sdk/core/azure-core/README.md

```diff
@@ -20,7 +20,7 @@ The main shared concepts of Azure::Core include:
 
 ### HTTP Transport adapter
 
-Out of the box, the Azure SDK for C++ supports the libcurl and WinHTTP libraries as HTTP stacks for communicating with Azure services over the network. The SDK also provides a mechanism for `customer-implemented` *HTTP transport adapter*.
+Out of the box, the Azure SDK for C++ supports the libcurl and WinHTTP libraries as HTTP stacks for communicating with Azure services over the network. The SDK also provides a mechanism for `customer-implemented` *HTTP transport adapter*. [Learn more](https://github.com/Azure/azure-sdk-for-cpp/blob/master/doc/HttpTransportAdapter.md#http-transport-adapter).
```

Comment:
Add the link to the links section at the bottom of the bottom and reference that link here.

Reply:
Isn't that useful only when the link is re-used?  Or why would we want to move all links to the bottom?
For one only link, IMHO, it makes looking at the link harder without getting any benefit